wiz-admission-controller/templates/serviceaccount.yaml

```diff
@@ -70,3 +71,38 @@ roleRef:
   name: {{ printf "%s-pull-secrets-reader" (include "wiz-admission-controller.serviceAccountName" .) }}
   apiGroup: rbac.authorization.k8s.io
 {{- end }}
+
+{{- if .Values.wizManager.enabled -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ printf "%s-pods-and-deployments-manager" (include "wiz-admission-controller.serviceAccountName" .) }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "wiz-admission-controller.labels" . | nindent 4 }}
+rules:
+  - apiGroups: ["apps"]
+    resources: ["deployments"]
+    resourceNames: {{ include "autoUpdate.deployments" . }}
+    verbs: ["get", "patch"]
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ printf "%s-pods-and-deployments" (include "wiz-admission-controller.serviceAccountName" .) }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "wiz-admission-controller.labels" . | nindent 4 }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "wiz-admission-controller.serviceAccountName" . }}
```

Comment:
Don't we want least privilege, to have another service account for the manager pod, so the ac pod won't be able now to "patch" deployments?